content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -262,12 +294,14 @@ Server:  Message Received
 ```
 Connected
 Client: Hey there 
-Client: am doing this for the first time
+Client: I am doing this for the first time
 Client: BYE
 Closing connection
 ```
- Important note
-Port number may cause an error if the port is already in use. To fix this, change the port number to a value that is not commercially owned.
+ Note of importance
```

Comment:
> Note of importance
> 
> If the port is already in use, the port number may result in an error. To solve the issue, change the port number to a value that isn't commercially licensed.

I would reformat this to be like so:

> \> Note of importance: If the port is already in use, the port number may result in an error. To solve the issue, change the port number to a value that isn't commercially licensed.

It would be nicer to have that as a blockquote so I would add a \> to make it a blockquote. In case you don't know what a blockquote looks like it's like this:

> this is a blockquote

 Also, I would probably put this all in the same paragraph and put a **:** after **"Note of importance"**.